server/game/cards/02_SHD/units/PunishingOne.ts

```diff
@@ -0,0 +1,30 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+
+export default class PunishingOne extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '8687233791',
+            internalName: 'punishing-one#dengars-jumpmaster',
+        };
+    }
+
+
+    public override setupCardAbilities() {
+        this.addTriggeredAbility({
+            title: 'You may ready this unit. Use this ability only once each round.',
```

Comment:
nit: we try to keep ability titles to just the part that describes the effect since this may be displayed in a handful of different contexts. We typically don't include limits or the "you may" at the front since there will be a pass button that indicates that. So in this case just `Ready this unit` is good